packages/db/src/project/test/compilationSources/.eslintrc.json

```diff
@@ -0,0 +1,27 @@
+{
+  "overrides": [
+    {
+      "files": ["./test/**/*.js"],
+      "env": {
+        "mocha": true
+      },
+      "globals": {
+        "interfaceAdapter": "readonly",
+        "web3": "readonly",
+        "assert": "readonly",
+        "expect": "readonly",
+        "artifacts": "readonly",
+        "config": "readonly",
+        "debug": "readonly",
+        "contract": "readonly"
+      }
+    },
+    {
+      "files": ["./migration/**/*.js"],
```

Comment:
All of these entries should be `./migrations/**/*.js`, right? They are currently all the singular form.

Reply:
Oh crap, missed that, thanks!